lang/golang/golang/Makefile

```diff
@@ -14,7 +14,8 @@ PKG_NAME:=golang
 PKG_VERSION:=$(GO_VERSION_MAJOR_MINOR)$(if $(GO_VERSION_PATCH),.$(GO_VERSION_PATCH))
 PKG_RELEASE:=1
 
-GO_SOURCE_URLS:=https://dl.google.com/go/ \
+GO_SOURCE_URLS:=https://go.dev/dl/ \
```

Comment:
`https://go.dev/dl/go1.18.2.src.tar.gz` is a 302 redirect to `https://dl.google.com/go/go1.18.2.src.tar.gz` so this is unnecessary.

Reply:
ok I dropped it